grafana/Dockerfile

```diff
@@ -11,11 +11,14 @@ ENV GF_ANALYTICS_REPORTING_ENABLED=false \
     GF_SECURITY_DISABLE_GRAVATAR=true \
     GF_USERS_ALLOW_SIGN_UP=false \
     GF_USERS_DEFAULT_LANGUAGE=detect \
-    GF_DATE_FORMATS_USE_BROWSER_LOCALE=true \
     GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH=/dashboards_internal/home.json \
     DATABASE_PORT=5432 \
     DATABASE_SSL_MODE=disable
 
+# This experimental config option is temporarily disabled (incompatible with Grafana 11.3.1)
+# https://github.com/grafana/grafana/issues/95209
+# ENV GF_DATE_FORMATS_USE_BROWSER_LOCALE=true
```

Comment:
[date_formats].use_browser_locale is broken with Scenes-powered Dashboards. Will be reenabled when fixed.